### PR TITLE
Feature/sg 597 model readme

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: documentation/source/models.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+To instantiate a model, specify the model name and the number of classes desired.
+```python
+from super_gradients.training import models
+
+# Instantiate resnet18 with head supporting 100 classes
+default_resnet18 = models.get(model_name="resnet18", num_classes=100)
+```
+
+Every model name is accessible in to [model zoo](http://bit.ly/3EGfKD4),but can also be dynamically accessed through `super_gradients.common.object_names` for autocompletion
```
Comment (ofrimasad) [inline documentation/source/models.md:21]: please rephrase: 
All model names are available in the [model zoo]...

File: documentation/source/models.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+
+### II. Instantiating a pretrained model
+When loading a pre-trained model, SuperGradients also provides a pre-trained head by default. 
+The head's dimension is determined by the number of classes in the dataset used for training.
+
+If you're using a different dataset, you'll need to change the number of classes in the head. 
+This keeps all the pre-trained weights of the model intact, except for the head which will be new and untrained. 
+The model will not be able to predict accurately until fine-tuned.
```
Comment (ofrimasad) [inline documentation/source/models.md:37]: do we have the names of the datasets each model was pretrained on in our zoo? if so, please state this.

File: documentation/source/models.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        out = self._head(out)
+        return out
+
+model = CustomModel(backbone=backbone_resnet18)
+```
+
+
+### Playing with the model architecture parameters 
+
+All the SuperGradients model architectures can be parametrized using `arch_params`.
```
Comment (ofrimasad) [inline documentation/source/models.md:91]: please fix the phrasing: "All of SuperGradients..."

**Review by ofrimasad — CHANGES_REQUESTED** [review]
Looks great. a few minor phrasing comments. 
Please add a section for instantiating a model (including the arch_params) from a YAML.

File: documentation/source/models.md (hunk 3 above)
Comment (Louis-Dupont) [reply documentation/source/models.md]: Changed in 0774f85754e17e396c6c377aeca58cdf8b888dd7

File: documentation/source/models.md (hunk 2 above)
Comment (Louis-Dupont) [reply documentation/source/models.md:37]: Changed in 0774f85754e17e396c6c377aeca58cdf8b888dd7

File: documentation/source/models.md (hunk 1 above)
Comment (Louis-Dupont) [reply documentation/source/models.md]: Changed in 0774f85754e17e396c6c377aeca58cdf8b888dd7

File: documentation/source/models.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+## Model Zoo
```
[the remark this answers is not in the document]
Comment (Louis-Dupont) [reply documentation/source/models.md]: Changed in 0774f85754e17e396c6c377aeca58cdf8b888dd7

**Review by ofrimasad — APPROVED** [review]
LGTM. very nice

**Pull request merged**